UI/html/home.html

```diff
@@ -6,23 +6,197 @@
     <title>The-Mentor</title>
     <link rel="stylesheet" href="../css/main.css">
     <link rel="shortcut icon" href="../img/download (2).png" type="image/x-icon"/>
+    <style>
+        #containerm{
+            margin:0 auto;
```

Comment:
Use external styling for CSS, not in-line styling...

Reply:
Thanks, I finished with it.
Thanks